translations/frontend/en.json

```diff
@@ -2561,7 +2561,8 @@
                         "info": "INFO",
                         "warning": "WARNING"
                     },
-                    "load_full_log": "Load Full Home Assistant Log",
+                    "load_full_log": "Show Full Home Assistant Log",
+                    "download_full_log": "Download Full Home Assistant Log",
```

Comment:
Only change the `en.json` file in the `src/translations` folder. The rest will be updated automatically via the translation scripts.

Reply:
thx for that hint